docs/examples/adr-pvarray/plot_fit_to_matrix.py

```diff
@@ -0,0 +1,110 @@
+"""
+Obtaining ADR model parameters from IEC 61853 matrix measurements
+=================================================================
+
+There's a fitting function provided in pvlib to do exactly that.
+
+Since PV module efficiency varies with irradiance and temperature
+what better way to train a model than using efficiency measurement
+over a broad range of temperature and irradiance levels?
+The standard IEC 61853-1 defines a standard matrix of conditions
+for such measurements and this example shows how the ADR model
+parameters can be determined with just a few lines of code using
+functions in pvlib-python.
+
+Author: Anton Driesse
+"""
+
+from io import StringIO
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from pvlib.pvarray import pvefficiency_adr, fit_pvefficiency_adr
+
+# %% The text on this line is not displayed
+#
+# Here are some matrix measurements:
+#
+
+iec61853data = '''
```

Comment:
Nit: this data set has more conditions than specified in IEC61853. Maybe give the data set a different name since the name appears in the example.

Reply:
I will drop the extra points.